models/clinicModel.js

```diff
@@ -18,7 +18,7 @@ const clinicSchema = new mongoose.Schema({
     max: 5,
     min: 0,
   },
-  phoneNumber: {
+  phoneNumbers: {
```

Comment:
Here in the clinic, it's good to add multiple numbers, but we need to extend the validators to include landline numbers

Reply:
OK i will fix Right now